AgentDiscoveries-Backend/src/main/java/org/softwire/training/api/routes/v1/TokenRoutes.java

```diff
@@ -50,7 +50,8 @@ public TokenResponseApiModel createToken(Request req, Response res) {
             User user = userOptional.get();
 
             if (passwordHasher.checkPassword(tokenRequest.getPassword(), user.getHashedPassword())) {
-                return generateToken(user);
+                TokenResponseApiModel token = generateToken(user);
```

Comment:
Don't think this needs changing, but note we originally only made this change to make debugging easier - try to consider whether changes are intended!